examples/bigcommerce/lib/bigcommerce/bigcommerce.ts

```diff
@@ -0,0 +1,195 @@
+const STORE_NAME = process.env.BIGCOMMERCE_STORE_NAME
+const ACCESS_TOKEN = process.env.BIGCOMMERCE_ACCESS_TOKEN
+
+const getBearerToken = async () => {
+  const A_WEEK_FROM_NOW = Math.floor(
+    new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * 7).getTime() / 1000,
+  )
+
+  const response = await fetch(
+    `https://api.bigcommerce.com/stores/${process.env.BIGCOMMERCE_STORE_HASH}/v3/storefront/api-token`,
```

Comment:
We probably want to be consistent in how we handle environment variables and configuration. I see that some are defined in module scope while others are defined in function scope. Also some of them are checked and others are not.

I would suggest making a `lib/config.ts` file that handles all configuration values and exports them in a clean interface that can then be used by other modules that need that configuration